### ISSUE TITLE
Request: Stavanger, Norway

### ISSUE THREAD

**Issue by sjabby** [mampfes/hacs_waste_collection_schedule#662]
Would it be possible to add support for the Collection Schedule for Stavanger, Norway?
Cant find any ical, rss, or data source. Maybe scraping?
Search for address: https://www.stavanger.kommune.no/renovasjon-og-miljo/tommekalender/finn-kalender/
Valid address: https://www.stavanger.kommune.no/renovasjon-og-miljo/tommekalender/finn-kalender/show?id=962f10fe-63fd-4d2d-90ed-663fbb6de7b2&municipality=Stavanger&gnumber=58&bnumber=662&snumber=1

App: https://play.google.com/store/apps/details?id=com.norconsult.tommeplan.stavanger&hl=no&gl=US&pli=1

**Comment by sjabby** [comment]
Sample address for the search: Ledaalsgata 10

**Comment by bbr111** [comment]
First tests looks good.

You need the ID, municipality, gnumber,bnumber and snumber as parameter.
the content html can be parsed. Will finish this tomorrow.

```
16.02 - torsdag
Restavfall
23.02 - torsdag
Bio
02.03 - torsdag
Restavfall
09.03 - torsdag
Papp/papir
Bio
16.03 - torsdag
Restavfall
23.03 - torsdag
Bio
30.03 - torsdag
Restavfall
04.04 - tirsdag
Papp/papir
Bio
13.04 - torsdag
Restavfall
20.04 - torsdag
Bio
27.04 - torsdag
Restavfall
04.05 - torsdag
Papp/papir

```

**Comment by bbr111** [comment]
@sjabby 

```
ICON_MAP = {
    "Restavfall": "mdi:trash-can",
    "Papp/papir": "mdi:recycle",
    "Bio": "mdi:leaf",
}
```

Could you please verify the ICON_MAP?

**Comment by bbr111** [comment]
@sjabby 

check if PR #665 need some changes

**Comment by sjabby** [comment]
ICON_MAP looks good, added a comment in the PR for Christmas trees as there is a yearly collection for that, normally in january.

**Comment by bbr111** [comment]
already changed the PR.

**Comment by sjabby** [comment]
Perfect, then it should be good.

**Comment by mampfes** [comment]
fixed by #665
